firestore-bigquery-export/guides/GENERATE_SCHEMA_VIEWS.md

```diff
@@ -314,6 +314,62 @@ Since all document data is stored in the schemaless changelog, mistakes in
 schema configuration don't affect the underlying data and can be resolved by
 re-running the schema-views script against an updated schema file.
 
+### Using Gemini to generate draft schema files
+
+Instead of manually creating schema files, you can use the built-in Gemini AI Agent to automatically analyze your Firestore collection and generate an appropriate schema. The agent will:
```

Comment:
```suggestion
Instead of manually creating schema files, you can use the built-in Gemini Agent to automatically analyze your Firestore collection and generate an appropriate schema. The agent will:
```